scripts/base/frameworks/telemetry/main.zeek

```diff
@@ -0,0 +1,498 @@
+##! Module for recording and querying metrics. This modules wraps
+##! the lower-level telemetry.bif functions.
```

Comment:
Suggest to add a note here about these being accessible via Prometheus as well, easy to miss that link.